includes/Plugin_Context.php

```diff
@@ -42,7 +43,13 @@ class Plugin_Context {
 	 * @param string $main_file The absolute path to the plugin main file.
 	 */
 	public function __construct( $main_file ) {
-		$this->main_file = wp_normalize_path( $main_file );
+		if ( function_exists( '\WP_CLI\Utils\normalize_path' ) ) {
+			$this->main_file = normalize_path( $main_file );
+		} elseif ( function_exists( 'wp_normalize_path' ) ) {
```

Comment:
Nit-pick: I think the order should be reversed since WP core is more "canonical" than WP-CLI :)